.github/actions/setup-linux/action.yml

```diff
@@ -45,3 +45,4 @@ runs:
       shell: bash
       run: |
         env | grep '^GITHUB' > "/tmp/github_env_${GITHUB_RUN_ID}"
+        env | grep '^CI' > "/tmp/github_env_${GITHUB_RUN_ID}"
```

Comment:
Confirming my understanding: this is where the variable `CI` is set, and the goal of this PR is to replace all IN_CI refs with CI and remove the explicit passing we do into the Docker images.

Reply:
No, the `CI` variable is set by the CI provider: 
- [Github](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables)
- [Circle](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables)

Here, the CI variable is written to an `--env-file` that we pass to docker, which I think we should do in general instead of passing env vars through manually.